src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,40 @@
+package shop;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CartTest {
```

Comment:
Please add one more test for Cart class, scenario will be like:
1. Add item
2. Remove item
3. Verify that removal was correct